src/mumble/VoiceRecorder.cpp

```diff
@@ -277,6 +289,15 @@ bool VoiceRecorder::ensureFileIsOpenedFor(SF_INFO &soundFileInfo, boost::shared_
 		return false;
 	}
 
+	//Enable constant 320kbps for MP3 formats
+	if (m_config.recordingFormat == VoiceRecorderFormat::MP3)
+	{
+		int bitrate = SF_BITRATE_MODE_CONSTANT;
+        double compression = 0.0;
+        sf_command(ri->soundFile, SFC_SET_BITRATE_MODE, &bitrate, sizeof(int));  
+        sf_command(ri->soundFile, SFC_SET_COMPRESSION_LEVEL, &compression, sizeof(double));
+	}
+
```

Comment:
Spacing seems to be inconsistent here. You can use ``clang-format`` (in version 10 !) to automagically format your code changes.